src/config/mod.rs

```diff
@@ -58,6 +58,9 @@ create_config! {
     chain_width: usize, 60, true, "Maximum length of a chain to fit on a single line.";
     single_line_if_else_max_width: usize, 50, true, "Maximum line length for single line if-else \
         expressions. A value of zero means always break if-else expressions.";
+    single_line_let_else_max_width: usize, 50, true, "Maximum line length for single line \
+        let-else statements. A value of zero means always format the divergent `else block` \
```

Comment:
nit but I don't think we want to capture the "block" word as part of the code fence:

```suggestion
        let-else statements. A value of zero means always format the divergent `else` block \
```